addons/mail/static/src/discuss/core/common/discuss_core_common_service.js

```diff
@@ -35,6 +35,7 @@ export class DiscussCoreCommon {
                     type: "info",
                 });
             }
+            thread.closeChatWindow();
```

Comment:
Would take opportunity of this IMP to reword `You unsubscribed from %s.` to `You left %(channel)s.`